Updated Manage Students UI

breity opened the pull request:
## Changes
- Updated Manage Students UI, styles, and layout.
- Teams and draggable students design matches Match student component visuals.
- Standardized dialog styles and layout.
- Show confirmation messages when change team period and move student requests complete successfully.

## Test
Make sure Manage Students functionality works as before:
- Teams and students all show up correctly.
- Empty teams show up at bottom of period.
- Moving students to new team and changing team periods work.
- Viewing student info works.

Closes #276.

breity committed:
Updated Manage Student UI, styles, and layout.
Created ManageTeams component.
Updated some text for consistency.
#276

breity committed:
Updated global layout style to better support wider screens. #276

breity committed:
Merge branch 'manage-students-rewrite' into issue-276-update-manage-students-styles

breity committed:
Added confirmation messages for manage students:
- Changing team's period
- Moving student to new team
Fixed show error message on http errors.

breity committed:
Added instructions for moving students.
Minor style fixes.
#276

breity committed:
Show student names as list in change team period dialog. #276

breity committed:
Added scroll while dragging support to Manage Students.
Fixed tests
#276

geoffreykwan reviewed (changes requested):
Overall looks and works well. I noticed some visual things that we could change.

- Change the text "Inactive Teams" to "Empty Teams"?
- In manage-period.component.html add i18n for "Tip: Drag student to change teams" and "Inactive Teams"
- In manage-team.component.html add i18n for "No Students"
- Style the student info popup field and values to be closer together?

breity committed:
Cleanup and text updates for Manage Students:
Changed 'Inactive Teams' to 'Empty Teams'.
Added missing i18n attributes.
Fixed missing <td> elements.
#276

geoffreykwan committed:
Add i18n for 'No students' text in manage-team.component.html. #276

geoffreykwan reviewed (approved):
Looks good.
